### PR TITLE
add TagBot

### PR DESCRIPTION
**Pull request by bjarthur**

### CODE REVIEW

**Comment by bjarthur** [comment]
a second commit changed CompatHelper frequency from every minute (doh!) to once per week.

**Comment by bjarthur** [comment]
a third commit changes CI to test on julia 1.3

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: CompatHelper
 
 on:
   schedule:
-    - cron: '* * * * 0'
+    - cron: '0 0 * * 0'
```
Comment (tlnagy) [inline .github/workflows/CompatHelper.yml]: Isn't this once per day?

Comment (bjarthur) [reply .github/workflows/CompatHelper.yml]: midnight on sunday, so once per week.  https://crontab.guru/#0_0_*_*_0

Comment (bjarthur) [reply .github/workflows/CompatHelper.yml]: it's a shame that both tagbot and compathelper poll.  would be nice if they could be re-engineered to use interrupts.

**Comment by bjarthur** [comment]
worth noting that the change to travis specifies "julia 1", not "julia 1.3".  so when 1.4 etc is released it will automatically start testing on it.  just learned this was possible from tim holy's work on indirectarrays.jl.

**Pull request merged**